brave/browser-laptop
# Refactor autofillAddressPanel.js and autofillCreditCardPanel.js

Closes #8099
Closes #8100
Addresses #7989

- Added CommonFormLarge
- Added cx to dialog.js for more readability
- Refactored with commonForm, replacing it with genericForm
  Aligned labels to the left and input textboxes to the right by applying display:flex
- Removed .manageAutofillDataPanel from form.less

Also:
- Edited autofillTest.js and autofill.js
  TODO: refactor autofill.js with Aphrodite
- Edited selectors.js

Auditors:

Test Plan:
1. Automated tests should pass
2. Open about:autofill
3. Add an address
4. Add a credit card
5. Visit https://www.roboform.com/filling-test-all-fields
6. Make sure the input forms can be filled
